README.md

```diff
@@ -13,42 +21,44 @@ Library for talking to the eos api.
 <script src='dist-web/eosjs2-jsonrpc-debug.js'></script>
 <script src='dist-web/eosjs2-jssig-debug.js'></script>
 <script>
-    let pre = document.getElementsByTagName('pre')[0];
-    let rpc = new eosjs2_jsonrpc.JsonRpc('http://localhost:8000');
-    let signatureProvider = new eosjs2_jssig.default(['5JtUScZK2XEp3g9gh7F8bwtPTRAkASmNrrftmx4AxDKD5K4zDnr']);
-    let api = new eosjs2.Api({ rpc, signatureProvider });
-
-    (async () => {
-        try {
-            let result = await api.pushTransaction({
-                blocksBehind: 3,
-                expireSeconds: 10,
-                actions: [{
-                    account: 'eosio.token',
-                    name: 'transfer',
-                    authorization: [{
-                        actor: 'useraaaaaaaa',
-                        permission: 'active',
-                    }],
-                    data: {
-                        from: 'useraaaaaaaa',
-                        to: 'useraaaaaaab',
-                        quantity: '0.0001 SYS',
-                        memo: '',
-                    },
-                }],
-            });
-            pre.textContent += '\n\nTransaction pushed!\n\n' + JSON.stringify(result, null, 4);
-        } catch (e) {
-            pre.textContent += '\nCaught exception: ' + e;
-            if (e instanceof eosjs2_jsonrpc.RpcError)
-                pre.textContent += '\n\n' + JSON.stringify(e.json, null, 4);
-        }
-    })();
+  let pre = document.getElementsByTagName('pre')[0];
+  const defaultPrivateKey = "5JmqocoeJ1ury2SdjVNVgNL1n4qR2sse5cxN4upvspU2R5PEnxP"; // thegazelle
```

Comment:
As long as testnet is intended to continue running, I think it would be beneficial to go ahead and create those test accounts, especially if one of our goals is to have DApp developers hit the ground running with an easy example.

Reply:
The testnet that `tutorials/bios-boot-tutorial/bios-boot-tutorial.py` in the eos repo creates has these accounts.

Reply:
Will the previously hardcoded private key always be the same for useraaaaaaaa?  I can change this back if this test will simply work after that one .py script has been run. If there is additional bootstrapping to make the test file work with useraaaaaaa, I think we would be better off using dedicated accounts on junglenet.

Reply:
It's always the same. The set of accounts is here: https://github.com/EOSIO/eos/blob/75635168e743c3e65bb6345a1eaedd905581e6b0/tutorials/bios-boot-tutorial/accounts.json . The script defaults to the first 3000, but I normally run it with the first 10 to speed things along.

Reply:
Perhaps we should include a link to the bios-boot-tutorial as one of the readme instruction points?  The reason why I changed this to begin with is because I didn't realize how easy it was to get it working with that python script.

Reply:
We should just add unit tests and nuke test.html unit tests can come in a later PR.  Lets work to get this PR in so we can move forward.